src-tauri/src/notification_manager.rs

```diff
@@ -0,0 +1,70 @@
+use notify_rust::Notification;
+use std::sync::LazyLock;
+
+use log::info;
+
+const LOG_TARGET: &str = "tari::universe::notification_manager";
+static INSTANCE: LazyLock<NotificationManager> = LazyLock::new(NotificationManager::new);
+
+pub enum CurrentOperatingSystem {
+    Windows,
+    Linux,
+    MacOS,
+}
```

Comment:
This enum exists in the `SystemtrayManager` can we define it in a single place and reuse it?